src/ophys_etl/utils/motion_border.py

```diff
@@ -48,20 +54,20 @@ def get_max_correction_values(x_series: pd.Series, y_series: pd.Series,
     down_shift = np.max(-1 * y_no_outliers.min(), 0)
```

Comment:
As we discussed in person, these should be simplified to remove the max if we actually want to return negative values. The zero here is specifying an axis on a scalar value.